### PR TITLE
add: `yt-dlp`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

File: packages/yt-dlp/yt-dlp.pacscript [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+build() {
+  make -j"$(nproc)"
+}
+
+install() {
+  sudo make install DESTDIR="${STOWDIR}/${name}"
+}
+
+postinst() {
+  yt-dlp -help
```
Comment (Henryws) [inline packages/yt-dlp/yt-dlp.pacscript:25]: Is this necessary?

Comment (WRM-42) [reply packages/yt-dlp/yt-dlp.pacscript:25]: No, it's only there for testing. I will remove it before merging

File: packages/yt-dlp/yt-dlp.pacscript [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+url="https://github.com/${name}/${name}/archive/refs/tags/${version}.tar.gz"
+build_depends="make pandoc zip python3-pytest"
+depends="python3 ffmpeg mpv python3-mutagen python3-pip python3-websockets python3-secretstorage python3-brotli python3-certifi python3-xattr atomicparsley rtmpdump"
+breaks="${name}-git ${name}-bin ${name}"
+description="A youtube-dl fork with additional features and fixes"
+hash="0e427a4df804787e1543a0ca42436564f0122d7a38b8712089ebd31cce39c224"
+maintainer="WRM-42 <y8bsbahy@anonaddy.me>"
+
+prepare() {
+  sudo pip3 install pycryptodomex
```
Comment (Henryws) [inline packages/yt-dlp/yt-dlp.pacscript:13]: This package should be removed with `removescript`

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — CHANGES_REQUESTED** [review]
Check CI logs

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `pkgname`

**Comment by WRM-42** [comment]
Why? `pkgname` is the same as `name`

**Comment by Henryws** [comment]
> Why? `pkgname` is the same as `name`

oh yeah, hehe, forgot.

**Review by Henryws — APPROVED** [review]

**Pull request merged**